admin/views/tabs/network/features.php

```diff
@@ -16,46 +16,48 @@
 $feature_toggles = Yoast_Feature_Toggles::instance()->get_all();
 
 ?>
-<h2><?php esc_html_e( 'Features', 'wordpress-seo' ); ?></h2>
-<div class="yoast-measure">
-	<?php
-	echo sprintf(
-		/* translators: %s expands to Yoast SEO */
-		esc_html__( 'This tab allows you to selectively disable %s features for all sites in the network. By default all features are enabled, which allows site admins to choose for themselves if they want to toggle a feature on or off for their site. When you disable a feature here, site admins will not be able to use that feature at all.', 'wordpress-seo' ),
-		'Yoast SEO'
-	);
+<div class="tab-block">
+	<h2><?php esc_html_e( 'Features', 'wordpress-seo' ); ?></h2>
+	<div class="yoast-measure">
+		<?php
+		echo sprintf(
+			/* translators: %s expands to Yoast SEO */
+			esc_html__( 'This tab allows you to selectively disable %s features for all sites in the network. By default all features are enabled, which allows site admins to choose for themselves if they want to toggle a feature on or off for their site. When you disable a feature here, site admins will not be able to use that feature at all.', 'wordpress-seo' ),
+			'Yoast SEO'
+		);
 
-	foreach ( $feature_toggles as $feature ) {
-		$help_text = esc_html( $feature->label );
-		if ( ! empty( $feature->extra ) ) {
-			$help_text .= ' ' . $feature->extra;
-		}
-		if ( ! empty( $feature->read_more_label ) ) {
-			$help_text .= ' ';
-			$help_text .= sprintf(
-				'<a href="%1$s" target="_blank" rel="noopener noreferrer">%2$s</a>',
-				esc_url( WPSEO_Shortlinker::get( $feature->read_more_url ) ),
-				esc_html( $feature->read_more_label )
-			);
-		}
+		foreach ( $feature_toggles as $feature ) {
+			$help_text = esc_html( $feature->label );
+			if ( ! empty( $feature->extra ) ) {
+				$help_text .= ' ' . $feature->extra;
+			}
+			if ( ! empty( $feature->read_more_label ) ) {
+				$help_text .= ' ';
+				$help_text .= sprintf(
+					'<a href="%1$s" target="_blank" rel="noopener noreferrer">%2$s</a>',
+					esc_url( WPSEO_Shortlinker::get( $feature->read_more_url ) ),
+					esc_html( $feature->read_more_label )
+				);
+			}
```

Comment:
Seems to me this whole `$help_text` is not used any longer. Removed in https://github.com/Yoast/wordpress-seo/commit/a0034a53d3c07c29af748abb22940f121f70cd66 

It was the help text used within the expandable help, e.g.:

<img width="656" alt="Screenshot 2020-04-28 at 13 59 32" src="https://user-images.githubusercontent.com/1682452/80485203-44f19880-8959-11ea-8d1d-c4e95652ac99.png">

If not used, please remove it.